checkov/terraform/checks/data/aws/GithubActionsOIDCTrustPolicy.py

```diff
@@ -4,54 +4,82 @@
                             continue
-                        principal_type = principal['type']
-                        principal_identifiers = principal['identifiers']
-                        if isinstance(principal_type, list) and len(
-                                principal_type) and 'Federated' in principal_type and isinstance(principal_identifiers,
-                                                                                                 list):
+                        principal_type = principal["type"]
+                        principal_identifiers = principal["identifiers"]
+                        if (
+                            isinstance(principal_type, list)
+                            and len(principal_type)
+                            and "Federated" in principal_type
+                            and isinstance(principal_identifiers, list)
+                        ):
                             for identifier in principal_identifiers:
-                                if isinstance(identifier,
-                                              list) and identifier[0] is not None and \
-                                        'oidc-provider/token.actions.githubusercontent.com' in identifier[0]:
+                                if (
+                                    isinstance(identifier, list)
+                                    and identifier[0] is not None
+                                    and "oidc-provider/token.actions.githubusercontent.com" in identifier[0]
+                                ):
                                     found_federated_gh_oidc = True
                                     break
                 if not found_federated_gh_oidc:
                     return CheckResult.PASSED
-                if found_federated_gh_oidc and not statement.get('condition'):
+                if found_federated_gh_oidc and not statement.get("condition"):
                     return CheckResult.FAILED
                 found_sub_condition_variable = False
                 found_sub_condition_value = False
-                for condition in statement.get('condition'):
-                    condition_variables = condition.get('variable')
-                    condition_values = condition.get('values')
+                for condition in statement.get("condition"):
+                    condition_variables = condition.get("variable")
+                    condition_values = condition.get("values")
                     if isinstance(condition_variables, list):
                         for condition_variable in condition_variables:
-                            if condition_variable == 'token.actions.githubusercontent.com:sub':
+                            if condition_variable == "token.actions.githubusercontent.com:sub":
                                 found_sub_condition_variable = True
                                 break
                         for condition_value in condition_values:
```

Comment:
```suggestion
                        if isinstance(condition_values, list):
                            for condition_value in condition_values:
```

Recommend checking that it is a list